MapboxDirections/MBRouteOptions.h

```diff
@@ -0,0 +1,32 @@
+#import <Foundation/Foundation.h>
+
+
+/**
+ The returned directions are appropriate for driving or riding a car, truck, or motorcycle.
+ 
+ This profile prioritizes fast routes by preferring high-speed roads like highways. A driving route may use a ferry where necessary.
+ */
+extern NSString *const MBDirectionsProfileIdentifierAutomobile;
```

Comment:
While you’re at it, can you make these constants into an extensible string enumeration?